src/main/java/util/Parser.java

```diff
@@ -0,0 +1,140 @@
+package util;
+
+import Commands.*;
```

Comment:
I think it is better to import classes from the `Commands` package one by one, instead of using `*` even though you indeed need all of them. In fact, that is what being specified by checkStyle.